api/Window.json

```diff
@@ -5709,6 +5709,153 @@
           }
         }
       },
+      "URL": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/URL",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "41"
+              },
+              {
+                "version_added": "8",
+                "version_removed": "41",
```

Comment:
Testing in a couple browsers myself, it doesn't look like the prefixes have actually been removed yet

Reply:
I've had tested on *console* window using this [link](https://turbo.net/run/chrome)
On v40 and the output was: *no deprication notice* 
On v41 it was: *depricated*

Reply:
I still think this line should be deleted. I tested it on Chrome 68 and got the deprecation warning, but `webkitURL` still works. Generally, we use `"version_removed"` to mean "this feature no longer works in this browser" even if the vendor discourages its use. So it hasn't been removed yet, at least for the purposes of our schema

Reply:
OK. I was under different assumptions. Got it now.